frontend/js/behaviors/core/stickySidebar.js

```diff
@@ -77,13 +77,6 @@ const stickySidebar = function(container){
         bottom();
       } else {
         sticky();
-
-        // Only add margin if the screen width is 1200px or above
-        if (window.innerWidth >= 1200) {
-          container.style.marginTop = (document.documentElement.classList.contains('p-digitalpublicationarticle-show') ? 0 : (!document.documentElement.classList.contains('s-unsticky-header') ? navContainer.clientHeight : 0) + (!document.documentElement.classList.contains('s-unsticky-digital-publication-header') ? stickyHeaderContainer.clientHeight : 0)) + 'px';
-        } else {
-          container.style.marginTop = '0px';
-        }
```

Comment:
Apparently, this wasn't even needed anymore, that's why the sidebar kept disappearing.

Reply:
Checked digital pubs sidebar and it seems to be performing as expected still with this code removed.